profiles/debug.lua

```diff
@@ -0,0 +1,99 @@
+-- Enable calling our lua profile code directly from the lua command line,
```

Comment:
all that's really missing is to implement the couple of c++ helper methods in lua